prow/gerrit/adapter/adapter_test.go

```diff
@@ -272,6 +298,14 @@ func TestProcessChange(t *testing.T) {
 					RunIfChanged: "\\.go",
 				},
 			},
+			{
+				JobBase: config.JobBase{
+					Name: "test-branch-pony",
+				},
+				Brancher: config.Brancher{
+					Branches: []string{"pony"},
```

Comment:
should we also test `SkipBranches` here?

Reply:
sure